Add sample code for loading ssn2 sessions into set using automation api

harveyAJ opened the pull request.

TimOliverMAT reviewed (commented):
Is there any reason why this sample code has been added as commented out code in the main method? I think it is going to be hard to find or understand from a users point of view. I think there should either be a specific example program that demonstrates loading a session, or at least create dedicated methods (not commented out) that work as almost self contained examples with a meaningful method name.

harveyAJ commented:
> Is there any reason why this sample code has been added as commented out code in the main method? I think it is going to be hard to find or understand from a users point of view. I think there should either be a specific example program that demonstrates loading a session, or at least create dedicated methods (not commented out) that work as almost self contained examples with a meaningful method name.

Latest commit contains a change addressing your comment. It's all in the same sample code, not in a separated app.

TimOliverMAT commented on HelloWorld.csharp/Program.cs:
```diff
@@ -21,12 +21,16 @@ public static class Program
         /// </summary>
         /// <prerequisites>
         ///     As a prerequisite this example expects ATLAS 10 to  be running, and a Session to be loaded into Set 1.
+        ///     Sample code (commented out) was added as an example on how to load a session into a set (SQLServer or SQLite sessions alike)
```
This comment is no longer correct

harveyAJ replied:
Fixed.

TimOliverMAT commented on README.md:
```diff
@@ -6,12 +6,16 @@ ATLAS Automation API allows an external application to control an instance of AT
+Both sample codes do the same<sup>1</sup>: creating a transient parameter, adding a Waveform Display, then `vCar` and the new parameter to it. 
 
-**Instructions**: To run start ATLAS, load a session with parameter vCar in it and execute the sample application. 
+**Instructions**: Before running, start ATLAS, load a session<sup>1</sup> with parameter `vCar` in it and execute the sample application<sup>2</sup>. 
```
The sample will now always load a session, is it still necessary for a session to be loaded as a prerequisite?

harveyAJ replied:
Still necessary for the Matlab code sample, as I've not made the change in there. Hence superscript referencing to a note in the README
